### ISSUE TITLE
SyntaxError @stripe/stripe-react-native/src/functions.ts

### ISSUE THREAD

**Issue by cnhuz** [stripe/stripe-react-native#1102]
When I start the project after adding @stripe/stripe-react-native@0.16.0, I get this error

```

SyntaxError: ../node_modules/@stripe/stripe-react-native/src/functions.ts: Unexpected token, expected ")" (60:16)
 58 |       paymentMethod: paymentMethod!,
  59 |     };
> 60 |   } catch (error: any) {
     |                 ^
  61 |     return {
  62 |       error,
  63 |     };

```

I modify the file all `catch (error: any)` to `catch(error)`, It works fine. In addition to this method, is there any way to avoid the error

**Comment by charliecruzan-stripe** [comment]
Closing since this isn't a bug in the SDK, but happy to help further where I can 

I think this has to do with your local setup, since it should be relying on the distributed JS files, rather than the TS files. Do you have typescript enabled in your project? 

You might need to update `metro-react-native-babel-preset` and `run yarn start --reset-cache`

**Comment by azuzi** [comment]
@charliecruzan-stripe 
Do you have documentation about the min version for ```metro-react-native-babel-preset``` ?

**Comment by rvera** [comment]
@cnhuz I ran into the same issue, the README stating that you need Babel `7.9.0` seems to be wrong, I was on `7.9.0` and solved this by bumping this to `7.14.0`. The project seems to be using `7.13.0` for the transform script plugin https://github.com/stripe/stripe-react-native/blob/master/package.json#L42

**Comment by AlehanderVoronin** [comment]
I was helped to solve this problem - update the version of react. Was 16.13.1 became 17.0.1
